src/lib/core/Optional.h

```diff
@@ -211,6 +212,12 @@ class Optional
     bool operator==(const T & other) const { return HasValue() && Value() == other; }
     bool operator!=(const T & other) const { return !(*this == other); }
 
+    std::optional<T> std_optional() const
```

Comment:
not an operator on purpose to make all casts explicit, hoping that in time we are able to convert everything and do not rely on implicit casts (and then we get full incremental updates).